lib/commander/index.js

```diff
@@ -527,14 +527,11 @@ Command.prototype.executeSubCommand = function(argv, args, unknown) {
 
   // In case of globally installed, get the base dir where executable
   //  subcommand file should be located at
-  var baseDir,
-    link = fs.lstatSync(f).isSymbolicLink() ? fs.readlinkSync(f) : f;
+  var baseDir;
```

Comment:
This file's contents were copied verbatim from https://github.com/tj/commander.js/blob/v2.20.0/index.js.